src/app.js

```diff
@@ -1 +1,238 @@
+            landName.appendChild(countryName);
+            landName.appendChild(population);
+            landName.appendChild(region);
+            landName.appendChild(flags);
+
+            switch (country.region) {
+                case  'Africa': countryName.style.color = 'blue'
+                    break;
+                case 'Asie': countryName.style.color = 'red'
+                    break;
+                case 'Oceania' : countryName.style.color = 'purple'
+                    break;
+                case 'Europe' : countryName.style.color = 'yellow'
+                    break;
+                default: countryName.style.color = 'green'
+
+            }
+
+        })
+        // console.log(response.data);
+    } catch (error) {
+
+        // Verwijzing naar error message
+        const errorMessage = document.getElementById('error-message');
+
+        // Check welke error message van toepassing is
+        if (error.response.status === 404) {
+            errorMessage.textContent = "Page Not Found | 404"
+        }
+        if (error.response.status === 500) {
+            errorMessage.textContent = "Internal Server Error | 500"
+        }
+    }
+
+}
+banaan();
+
+//-----------------------------------------EINDE OPDRACHT 1---------------------------------------//
+
+
+//------------------------------------- OPDRACHT 2-----------------------------------------------//
+
+
+const countryP2 = document.getElementById("country-part2")
+let errorMessage = "";
+
+async function fetchSpecData(country){
+
+    try {
+        const response = await axios.get(`https://restcountries.com/v2/name/${country}`,{
+            params : {
+                userInput : country
+            }
+        });
+
+        countryP2.replaceChildren();
+        errorMessage.replaceChildren();
+
+
+
+        response.data.map((country) => {
+
+            const {flags:{png}, name, capital, subregion, population, currencies,languages
+            } = country;
+
+            let coinString
+            currencies.map((coin) => {
+                if (currencies.length < 2) {
+                    coinString = `and you can pay with ${coin.name}`
+                } else {
+                    for (let i = 0; i < currencies.length; i++) {
+                        coinString = `and you can pay with ${coin.name[0]} and ${coin.name[1]}`
+                    }}})
+            let soundOfTalks
+            const soundOfTalksArray = languages.map((sound)=> {
+                return sound.name;
+            })
+                    if (languages.length < 2){
+                        soundOfTalks = `They speak ${soundOfTalksArray[0]}`
+                    }else if ( languages.length === 2) {
+                        soundOfTalks = `They speak ${soundOfTalksArray[0]} and ${soundOfTalksArray[1]} `
+                    }else {
+                       soundOfTalks = `They speak ${soundOfTalksArray[0]} , ${soundOfTalksArray[1]} and ${soundOfTalksArray[2]} `
+                    }
```

Comment:
Dit is niet echt uitbreidbaar als er meerdere languages zouden komen. Je kan hier gebruikmaken van een for-loop of de [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)